### ISSUE TITLE
斐讯K2路由器如何安装Python

### ISSUE THREAD

**Issue by 15757170756** [xyuanmu/XX-Mini#95]
活动买了一个斐讯K2路由器，想用xx-net翻墙，感觉先要在路由器上面安装Python才行的

**Comment by jzp820927** [comment]
先刷固件

**Comment by 15757170756** [comment]
我刷了华硕的固件。RT-AC54U，版本3.4.3.9-099_07-17，我看了看tmp中还有22M的存储量应该够的吧。可以实现adbyby功能，看优酷爱奇艺视频没有广告，这个比较傻瓜，一键就可以实现。但是迅雷快鸟和翻墙还有免流都还没有实现。如果能在路由器上实现翻墙还是很爽的；还有就是免流，架设OpenVPN，听着就酷炫，但是还没有实现。不知大神能否教一教。

**Comment by jzp820927** [comment]
这个路由器，你想弄这么多东西，你不想哈，CPU 吃不吃得消

**Comment by 15757170756** [comment]
对 是的。K2路由器的运行内存也不高，话说屏蔽广告已经可以了。然后翻墙的话，运行在Windows上的XX_NET也是可以的，所以如果再能实现免流也是极好的。请问大神有没有教程方法？
